tensorflow/c/kernels/BUILD

```diff
@@ -48,6 +70,40 @@ tf_cc_test(
     ],
 )
 
+tf_cc_test(
+    name = "summary_op_test",
+    srcs = ["summary_op_test.cc"],
+    deps = [
+        ":summary_op",
+        "//tensorflow/c:kernels",
+        "//tensorflow/core:framework",
+        "//tensorflow/core:lib",
+        "//tensorflow/core:test",
+        "//tensorflow/core:test_main",
+        "//tensorflow/core:testlib",
+    ],
+)
+
+cc_library(
+    name = "tensor_shape_utils",
+    srcs = ["tensor_shape_utils.cc"],
+    hdrs = ["tensor_shape_utils.h"],
+    deps = [ "//tensorflow/c:tf_tensor"], 
+    visibility = ["//visibility:private"],
+)
+
+tf_cc_test(
+    name = "tensor_shape_utils_test",
+    srcs = ["tensor_shape_utils_test.cc"],
+    deps = [
+        "//tensorflow/core:framework",
+        ":tensor_shape_utils",
```

Comment:
The correct ordering here is:


        ":tensor_shape_utils",
        "//tensorflow/core:framework",
        "//tensorflow/core:lib",
        "//tensorflow/core:test",
        "//tensorflow/core:test_main",